libraries/Microsoft.Bot.Builder.AI.LUIS/Schemas/Microsoft.LuisRecognizer.schema

```diff
@@ -81,34 +86,34 @@
             "description": "Options to control LUIS prediction behavior.",
             "properties": {
                 "includeAllIntents": {
+                    "$ref": "schema:#/definitions/booleanExpression",
                     "type": "boolean",
                     "title": "Include all intents",
```

Comment:
You should probably remove all of these types--you get them from the included expression.

Reply:
removed